src/frontend/screens/Settings/MapSettings/BackgroundMaps.tsx

```diff
@@ -0,0 +1,180 @@
+import { BottomSheetModalMethods } from "@gorhom/bottom-sheet/lib/typescript/types";
+import * as React from "react";
+import * as DocumentPicker from "expo-document-picker";
+import { defineMessages, FormattedMessage, useIntl } from "react-intl";
+import { ScrollView, StyleSheet, Text } from "react-native";
+import { NavigationStackScreenComponent } from "react-navigation-stack";
+import { MEDIUM_GREY } from "../../../lib/styles";
+import { BGMapCard } from "../../../sharedComponents/BGMapCard";
+import {
+  BottomSheetContent,
+  BottomSheetModal,
+  useBottomSheetModal,
+} from "../../../sharedComponents/BottomSheetModal";
+import Button from "../../../sharedComponents/Button";
+import HeaderTitle from "../../../sharedComponents/HeaderTitle";
+import Loading from "../../../sharedComponents/Loading";
+
+const m = defineMessages({
+  addBGMap: {
+    id: "screens.Settings.MapSettings.BackgroundMaps",
+    defaultMessage: "Add Background Map",
+  },
+  close: {
+    id: "screens.Settings.MapSettings.close",
+    defaultMessage: "Close",
+  },
+  importFromFile: {
+    id: "screens.Settings.MapSettings.importFromFile",
+    defaultMessage: "Import from File",
+  },
+  title: {
+    id: "screens.Settings.MapSettings.title",
+    defaultMessage: "Background Maps",
+  },
+  noAreas: {
+    id: "screens.Settings.MapSettings.noAreas",
+    defaultMessage: "No Downloaded Offline Area",
+    description:
+      "Message to indicate to user that no offline areas have been downloaded",
+  },
+});
+
+interface BackgroundMap {
+  size: number;
+  title: string;
+  mapId: string;
+}
+
+export const BackgroundMaps: NavigationStackScreenComponent = ({
+  navigation,
+}) => {
+  const { closeSheet, isOpen, openSheet, sheetRef } = useBottomSheetModal({
+    openOnMount: false,
+  });
+
+  const [backgroundMapList, setBackgroundMapList] = React.useState<
+    BackgroundMap[]
+  >();
+
+  React.useEffect(() => {
+    //To do: get background map list from server
+    setBackgroundMapList([]);
+  }, []);
+
+  const { formatMessage: t } = useIntl();
+
+  return (
+    <React.Fragment>
+      <ScrollView style={styles.container}>
+        <Button style={[styles.button]} variant="outlined" onPress={openSheet}>
+          {t(m.addBGMap)}
+        </Button>
+
+        {/* Default BG map card */}
+        <BGMapCard
+          mapId="D3f4U1t"
```

Comment:
probably fine to make this something readable e.g. `mapeo-default`

Reply:
This is going to change and be dynamically added, so its just a place holder for now